src/nl/codefox/contextvh/tygronEvents.mod2g

```diff
@@ -65,6 +65,19 @@ module tygronEvents {
 	forall percept(action_logs(ActList)), 
 		bel(member(actionlog(StakeholderID,Description,ActionID,IncList),ActList))
 		do insert(actionlog(StakeholderID,Description,ActionID,IncList)).
+		
+	%%% RELEVANT AREAS PERCEPTS
+	% insert relevant_areas/2
+	if percept(relevant_areas(CallID,MultiPolygons)), not(bel(relevant_areas(CallID,_)))
+		then insert(relevant_areas(CallID,MultiPolygons)).
+	% update relevant_areas/2
+	if percept(relevant_areas(CallID,NewMultiPolygons)), bel(relevant_areas(CallID,OldMultiPolygons))
+		then delete(relevant_areas(CallID,OldMultiPolygons))+ insert(relevant_areas(CallID,NewMultiPolygons)).
+		
+	%%% REQUEST PERCEPTS
+	% insert request/9 to beliefs
+	forall percept(request(Type,RequestType,RequestID,ContentLinkID,VisibieStakeholdersIDList,ActionLogIDList,Price,MultiPolygon,Area,Answers)) 
+		do insert(request(Type,RequestType,RequestID,ContentLinkID,VisibieStakeholdersIDList,ActionLogIDList,Price,MultiPolygon,Area,Answers)). 
```

Comment:
The `%%%REQUEST PERCEPT` can be removed completely, this is being handled by tygronRequests, if you remove line 77-80 then you are done.